index.js

```diff
@@ -0,0 +1,64 @@
+import * as LUXON from 'luxon';
```

Comment:
- [ ] due to Luxon not loading correctly your application is not working, the user is not able to change between the different windows.